server/libs/Controller.php

```diff
@@ -58,6 +58,9 @@ public static function request($key, $secure = false) {
 
         if($secure) {
             $config = HTMLPurifier_Config::createDefault();
+            $config->set('HTML.SafeIframe', true);
+            $config->set('URI.SafeIframeRegexp', '%^(https?:)?//(www\.youtube(?:-nocookie)?\.com/embed/|player\.vimeo\.com/video/|psv4\.userapi\.com/|.*drive\.google\.com)%');
```

Comment:
What is this `psv4\.userapi\.com/` domain, is it part of vk?

Reply:
Yes it is. However should be postponed cause V.I. uses dynamic hosting :(